subiquitycore/tui.py

```diff
@@ -234,9 +238,13 @@ async def _move_screen(self, increment, coro):
                 return
 
     async def move_screen(self, increment, coro):
-        view = await self.wait_with_progress(
+        view_or_callable = await self.wait_with_progress(
             self._move_screen(increment, coro))
-        if view is not None:
+        if view_or_callable is not None:
+            try:
```

Comment:
In client.py we check with `callable`.  I'm sure either is fine but I would like consistency.   If there is no other criteria, we could choose the faster one.

Reply:
No problem. I will go with `callable()` in both places then.
Using the `try ... except` in the other function would result in the callable actually being called needlessly just for the sake of testing if it is callable. I'd rather avoid that.